contributors/devel/sig-testing/writing-good-conformance-tests.md

```diff
@@ -0,0 +1,56 @@
+# Writing Good Conformance Tests for Kubernetes #
+
+Conformance tests are promoted from existing e2e tests which have
+been proven to be stable and "non-flaky".
+
+Please read the the 
+["Writing good e2e
+tests"](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md)
+guide as a supplment to this document.
+
+## Patterns and Anti-Patterns ##
+
+### Goals of Conformance tests ###
+
+The overarching goal of conformance tests is to exercise core
```

Comment:
I would lead with this paragraph b/c it outlines purpose.

Reply:
@timothysc, what do you think of the changes in https://github.com/kubernetes/community/pull/4107/commits/7acd706ee729602e6de464364107e4db773bdd55 ?